src/Mono.Android/Android.Runtime/Logger.cs

```diff
@@ -58,18 +61,28 @@ internal static class Logger {
 					System.Console.WriteLine ("[{0}] {1}: {2}", level, appname, line);
 			}
 		}
+
+		[DllImport ("__Internal")]
+		extern static uint monodroid_get_log_categories ();
+
+		static Logger ()
+		{
+			lock (initLock) {
```

Comment:
Is this strictly necessary?  Static constructors will only ever be executed once, so this seems redundant.